src/components/GoogleCalendarButton.js

```diff
@@ -26,41 +26,90 @@ export default function GoogleCalendarButton() {
     Mo: '2020-04-06T',
   };
   const events = [];
-  const handleClick = () => {
-    sections.forEach(
-      (section) => {
-        // '2015-05-28T09:00:00-07:00'
-        const dow = section.event.dow;
-        const date = dateOfTheFirstClass[dow];
-        const startTime = `${date + section.event.start.hour}:${section.event.start.minute}:00${timeZone}`;
-        const endTime = `${date + section.event.end.hour}:${section.event.end.minute}:00${timeZone}`;
-        const event = {
-          summary: `${section.subjectId}${section.courseId}`,
-          description: section.name,
-          location: section.event.location,
-          start: {
-            dateTime: startTime,
-            timeZone: 'America/Chicago',
-          },
-          end: {
-            dateTime: endTime,
-            timeZone: 'America/Chicago',
+  const clientID = '4436581685-mcdkhnh50a6di21k15do7d4a2orq7faf.apps.googleusercontent.com'
+  // const clientSecret = 'gPwoF99zNeSrs_JXcVaNATub'
```

Comment:
Keep in mind this repo is open source, you can't push stuff like this up even if it's commented out. Can you generate a new one?

Reply:
Ohh I didn't realize this was not supposed to be seen until now. Will get a new one

Reply:
Anything labeled "secret" is a good bet that we'll need to be careful with it.